client/state/selectors/test/get-post-revision-changes.js

```diff
@@ -116,12 +118,14 @@ describe( 'getPostRevisionChanges', () => {
 		const diff = getPostRevisionChanges( outOfOrderState, 12345678, 10, 16 );
 		expect( diff.content ).to.eql( [
 			{
+				count: 11,
```

Comment:
@marekhrabe -- not entirely sure why these weren't there before... sanity check?

Reply:
Not sure… It was probably ignored because we are not using that prop anywhere. Good idea to add it :)